scavenge/tutorial/03-scaffolding.md

```diff
@@ -0,0 +1,39 @@
+---
+order: 3
+---
+
+# Scaffolding
+
+We'll be using a CLI tool called [Starport](https://github.com/tendermint/starport) to create a new Cosmos SDK blockchain. To use `starport` [install it](https://docs.starport.network/intro/install.html) on your local machine:
+
+```
+curl https://get.starport.network/starport | bash
+```
+
+Scaffold anew Cosmos SDK blockchain using the `starport scaffold chain` command. The command accepts one argument: the Go module path that will be used for the project.
+
+By default a chain is scaffolded with a new empty Cosmos SDK module. Use the `--no-module` flag to skip module scaffolding.
+
+```
+starport scaffold chain github.com/cosmonaut/scavenge --no-module
+```
+
+This command created a new directory `scavenge` with a brand new Cosmos SDK blockchain. This blockchain doesn't have any application-specific logic yet, but it imports standard Cosmos SDK modules, such as `auth`, `bank`, `mint` and others.
+
+Change the current directory to `scavenge`:
+
+```
+cd scavenge
+```
+
+Inside the project directory you can execute other Starport commands to start a blockchain node, scaffold modules, messages, types, generate code, and much more.
+
+In a Cosmos SDK blockchain application-specific logic is implemented in separate modules. This keeps code easy to understand and reuse.
+
+Scaffold a new module called `scavenge`. Based on our design the `scavenge` module will be sending tokens between participants. Sending tokens is implemented in the standard `bank` module. Specify `bank` as a dependency using the optional `--dep` flag.
+
+```
+starport scaffold module scavenge --dep bank
```

Comment:
I haven't checked in the tutorial but FYI the `--dep` flag will "inject" the bank keeper in the scavenge keeper but you still need to specify the methods you need from bank in `types/expected_keeper.go` so we must specify this if it's not the case

Reply:
Yep! I mentioned it a bit later:

https://github.com/cosmos/sdk-tutorials/pull/751/files#diff-f32c80c9619ee1123ea6edf4db4412f6f55e95d1feda983b54885741534bd655R62-R65